xds/src/main/java/io/grpc/xds/XdsClientImpl.java

```diff
@@ -952,17 +960,13 @@ static VirtualHost findVirtualHostForHostName(
    * Response data for requested clusters is cached locally, in case of new cluster watchers
    * interested in the same clusters are added later.
    */
-  private void handleCdsResponse(DiscoveryResponse cdsResponse) {
-    if (logger.isLoggable(XdsLogLevel.DEBUG)) {
-      logger.log(XdsLogLevel.DEBUG, "Received CDS response:\n{0}", respPrinter.print(cdsResponse));
-    }
-    adsStream.cdsRespNonce = cdsResponse.getNonce();
-
+  private void handleCdsResponse(List<Any> resources, String versionInfo, String cdsRespNonce) {
```

Comment:
Because it updates `AdsStream.cdsRespNonce`.
https://github.com/grpc/grpc-java/blob/08557c9b215f14658ce4e121486080a1542f18a8/xds/src/main/java/io/grpc/xds/XdsClientImpl.java#L964

Reply:
Isn't that already updated in line 1458?